libass/ass_coretext.c

```diff
@@ -62,18 +62,21 @@ static void destroy_font(void *priv)
 
 static bool check_postscript(void *priv)
 {
+    bool ret = false;
     CTFontDescriptorRef fontd = priv;
     CFNumberRef cfformat =
         CTFontDescriptorCopyAttribute(fontd, kCTFontFormatAttribute);
     int format;
 
     if (!CFNumberGetValue(cfformat, kCFNumberIntType, &format))
-        return false;
+        goto fail;
```

Comment:
Instead of this `goto` I think a simple assignment of the `ret` variable under (inverse) condition is better.

Reply:
Sure.